spec/features/settings/dex_connector_ldap_feature_spec.rb

```diff
@@ -134,67 +134,51 @@
   describe "#new_save_disabled_by_change" do
     before do
       visit new_settings_dex_connector_ldap_path
-      have_button("Save", disabled: true)
+      expect(page).to have_button("Save", disabled: true)
 
       # Test that we can enable the button
       page.execute_script("$('#ldap_conn_save').removeProp('disabled')")
-      have_button("Save", disabled: false)
+      expect(page).to have_button("Save", disabled: false)
     end
 
     it "allows change of start_tls" do
-      # Is false by default - change to true
-      # Following line should work but it does not so using JS
-      # choose id: "dex_connector_ldap_start_tls_true"
-      page.execute_script("$('#dex_connector_ldap_start_tls_true').prop('checked', true)")
-      page.execute_script("$('#dex_connector_ldap_start_tls_true').trigger('change')")
+      find("#dex_connector_ldap_start_tls_true_label").click
```

Comment:
I do think this is less stable than using jquery. I understand it is the "rspec way" but I feel like it can break ( such as by removing the id from the labels )

Reply:
I agree that it feels fragile but what we did here is mimic what the user actually does on the page. The user goes to the page and click on the label "Yes" or "No" that is associated with the input radio.

> I still don't understand why the id has to be on the label instead of working on the actual input fields. That does not make sense to me. 

As explained above, it's just a matter of reproducing what the user actually does on the page.

The test could also be done using the `choose` or `click` directly on the input radio but it wouldn't be 100% true to what is done by the user in the end, which is what we are trying to achieve.

> I would like to know if a full manual test has been done to ensure the page still works from a sanity perspective.

Yes, it was done.